ls1intum/Artemis
# [Bugfix] Fix flaky test by storing feedbacks in the correct way

### Checklist

### Motivation and Context


`de.tum.in.www1.artemis.programmingexercise.ProgrammingExerciseResultJenkinsIntegrationTest.shouldGenerateNewManualResultIfManualAssessmentExists(ProgrammingExerciseResultJenkinsIntegrationTest.java:89)`

The problem is explained in detail here: https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/database/?highlight=%40OrderColumn#best-practices. When saving an ordered collection, you basically have to first save the collection entities first on their own before saving the whole collection.

### Description

Fixed the problem by storing the feedback in the correct way

### Steps for Testing

Go through the whole workflow of an programming exercise and check that everything works (create pe, take part in ce and get an automatic result, change the automatic result in the manual assessment)

### Test Coverage



- no changes
### Screenshots


- no changes
